docs/tutorialkit.dev/src/content/docs/reference/configuration.mdx

```diff
@@ -170,3 +177,30 @@ Navigating to a lesson that specifies `autoReload` will always reload the previe
 Specified which folder from the `src/templates/` directory should be used as the basis for the code. See the "[Code templates](/guides/creating-content/#code-templates)" guide for a detailed explainer.
 <PropertyTable inherited type="string" />
 
+#### `editPageLink`
+Display a link in lesson for editing the page content.
+The value is a URL pattern where `:path` is replaced with the lesson's location relative to the `src/content/tutorial`.
```

Comment:
@Nemikolh is this location logic OK? It's using the `id` from Astros `entry.id` now.

Reply:
I think this is fine yeah. :+1: